parsl/executors/high_throughput/executor.py

```diff
@@ -224,6 +226,10 @@ class HighThroughputExecutor(BlockProviderExecutor, RepresentationMixin, UsageIn
         Parsl will create names as integers starting with 0.
 
         default: empty list
+
+    enable_ipv6: bool
+        Set this flag to enable communication over IPV6 in addition to IPV4.
+        default: False
```

Comment:
We may want to name this `enable_ipv6_loopback`, or something else that more accurately describes what the flag does. Alternatively, we could create a `loopback_address` argument and allow users to specify any address in `127.0.0.0/8` or `::1/128`. I prefer the latter option.

Reply:
this maybe doesn't need "enable" on the start. Name the feature thats being turned on or off, like `encrypted` or `worker_debug`.

use a lowercase `v` in IPv6 - see https://datatracker.ietf.org/doc/html/rfc8200